harvard-lts/Harvard-Geospatial-Library
# Bumping rack

**Bumps rack gem **
* * *

# What does this Pull Request do?
Fixes a rack notification (bumps the gem)

# How should this be tested?

Spin up local and make sure GBL comes up -- I also have this deployed to dev if you would like to test records there.

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests? n
- integration tests? n

# Interested parties
Tag (@ mention) interested parties
